projects/control-service/projects/base/src/main/java/com/vmware/taurus/security/SecurityConfiguration.java

```diff
@@ -59,6 +69,28 @@
     private final String customClaimName;
     private final Set<String> authorizedCustomClaimValues;
     private final Set<String> authorizedRoles;
+    public static final String[] ENDPOINTS_TO_IGNORE = {
+          "/",
+          "/data-jobs/v2/api-docs",
+          "/data-jobs/swagger-resources/**",
+//        "/data-jobs/configuration/**",
+          "/data-jobs/swagger-ui.html",
+          "/data-jobs/webjars/**",
+          // There should not be sensitive data in prometheus, and it makes
+          // integration with the monitoring system easier if no auth is necessary.
+          "/data-jobs/debug/prometheus",
+          // TODO: likely /data-jobs/debug is too permissive
+          // but until we can expose them in swagger they are very hard to use with Auth.
+          "/data-jobs/debug/**"};
+
+    @Value("${datajobs.security.kerberos.kerberosPrincipal}")
+    private String kerberosPrincipal;
+
+    @Value("${datajobs.security.kerberos.keytabFileLocation}")
+    private String keytabFileLocation;
+
+    @Value("${datajobs.security.kerberos.enabled:false}")
```

Comment:
Isn't better to pass them in the constructor as the rest of the options?

Reply:
Now that we have multiple authentication mechanisms, we should have a flag for oauth2 authentication as well (before it was switched on/off by the general security flag).

Reply:
But please add the flag in a separate (short) PR. This one is about Kerberos.

Reply:
https://github.com/vmware/versatile-data-kit/pull/765 But we should review this when we finish with this commit since it depends on code introduced here.